src/Containers/DropArea/DropArea.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import styled from 'styled-components';
+import { MainInterface, Main } from '@Utils/BaseStyles';
+
+export interface IDropAreaProps
+    extends MainInterface,
+        React.HTMLAttributes<HTMLDivElement> {
+    message?: string;
+    isDragEnter?: boolean;
```

Comment:
onDragEnter
onDragLeave
As functions instead

Reply:
but I am not using the dropzone component in this component. do you want me to use this library for this component (dropzone)?

Reply:
Yes you can use the drag part of the libarary

Reply:
I mean, to handle the files and all this...

Reply:
ok I will try and see what happens...

Reply:
No do not move the logic for handling files. It's just a DragEvent like I listed in Teams. 

Don't put workers here! The file can be dropped then just an action is emitted of file details that's it